gulpfile.js

```diff
@@ -92,11 +100,11 @@ gulp.task('watch', function () {
 		'src/**/*.js',
 		'src/**/*.scss'
 	], function () {
-		runSequence('build', 'html');
+		runSequence('clean', 'build', 'html');
```

Comment:
I assume this line is the watch task - do we need to clean every time? I'm assuming yes.

Q) Did you consider just specifying clean as a dependency of build? I've noticed none of our gulp tasks have required tasks defined - I wonder if this is a good practice because everything is isolated etc?